common/changes/xxsFacepile_2017-01-24-21-47.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Added new XXS Persona size, also usable in Facepile.. Made accessibility improvements to Persona and Facepile controls.",
```

Comment:
There are two periods after "Facepile". 

Separately, Might be better to have 2 "changes" documenting the change to Facepile: .... and Persona: .... individually as well.